app-games/src/main/java/com/aptoide/android/aptoidegames/feature_promotional/NewsBundle.kt

```diff
@@ -0,0 +1,143 @@
+package com.aptoide.android.aptoidegames.feature_promotional
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.extensions.PreviewDark
+import cm.aptoide.pt.feature_editorial.domain.ArticleMeta
+import cm.aptoide.pt.feature_editorial.presentation.ArticleListUiState
+import cm.aptoide.pt.feature_editorial.presentation.ArticleListUiStateProvider
+import cm.aptoide.pt.feature_editorial.presentation.rememberEditorialListState
+import cm.aptoide.pt.feature_home.domain.Bundle
+import cm.aptoide.pt.feature_home.domain.randomBundle
+import com.aptoide.android.aptoidegames.AptoideFeatureGraphicImage
+import com.aptoide.android.aptoidegames.editorial.buildEditorialRoute
+import com.aptoide.android.aptoidegames.feature_apps.presentation.SmallEmptyView
+import com.aptoide.android.aptoidegames.home.LoadingBundleView
+import com.aptoide.android.aptoidegames.theme.AGTypography
+import com.aptoide.android.aptoidegames.theme.AptoideTheme
+import com.aptoide.android.aptoidegames.theme.Palette
+
+@Composable
+fun NewsBundle(
```

Comment:
I would not name it NewsBundle since it is not a bundle, but a single element.
I added Promotional name on mine, like NewsPromotionalCard for example